testcli.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
```

Comment:
we follow the following import format for ease of readability

```
std library imports

3-rd party imports

imports from openshift-azure
```

run `make verify` on our repo to get a report of files which violate such formatting. Your IDE should be able to help you apply `goimports` directly to files

always run `make verify` after `make unit` before checking in any code to the repo. We'll check your push anyway in CI but I think this is a good habit to develop.